docs/data-sources/obs_buckets.md

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: "Object Storage Service (OBS)"
+---
+
+# flexibleengine_obs_buckets
+
+Use this data source to get all OBS buckets.
+
+```hcl
+data "flexibleengine_obs_buckets" "buckets" {
+  bucket = "your-bucket-name"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) The region in which to obtain the OBS bucket.
+  If omitted, the provider-level region will be used.
+
+* `bucket` - (Optional, String) The name of the OBS bucket.
+
+* `enterprise_project_id` - (Optional, String) The enterprise project id of the OBS bucket.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The ID of the list.
+
+* `buckets` - A list of OBS buckets.
+
+The `buckets` block supports:
```

Comment:
```markdown
* `buckets` - A list of OBS buckets. The [buckets](#obs_buckets) object structure is documented below.

<a name="obs_buckets"></a>
The `buckets` block supports:
```

This specification is suitable to the fields which have son-fields.

Reply:
![image](https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/assets/145660317/fc3c9e57-090f-4dee-af67-f8ef502140e7)
revised